src/main/java/uk/gov/gchq/palisade/exception/PalisadeRuntimeException.java

```diff
@@ -15,6 +15,8 @@
  */
 package uk.gov.gchq.palisade.exception;
 
+import uk.gov.gchq.palisade.Generated;
+
 import static uk.gov.gchq.palisade.exception.Status.INTERNAL_SERVER_ERROR;
 
 /**
```

Comment:
Use the setters available for the methods below

Reply:
See the comment for PalisadeCheckedException.